.github/workflows/clp-uv-checks.yaml

```
@@ -0,0 +1,50 @@
+      - "components/clp-mcp-server/pyproject.toml"
+      - "components/clp-mcp-server/uv.lock"
+      - "components/clp-package-utils/pyproject.toml"
+      - "components/clp-package-utils/uv.lock"
+      - "components/clp-py-utils/pyproject.toml"
+      - "components/clp-py-utils/uv.lock"
+      - "components/job-orchestration/pyproject.toml"
+      - "components/job-orchestration/uv.lock"
+      - "integration-tests/pyproject.toml"
+      - "integration-tests/uv.lock"
+  push:
+    paths: *monitored_paths
```

actionlint (1.7.8)

"paths" section must be sequence node but got alias node with "" tag

(syntax-check)